jq-static/tests/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/tlog/6/20181108151533
```

Comment:
```suggestion
#/ Example: test.sh core/jq-static/1.6/20181108151533
```

Reply:
Ah, I think I understand the problem here. I'll update.